src/rules/Rule_4_10_2.cpp

```diff
@@ -0,0 +1,46 @@
+//===-  Rule_4_10_2.cpp - Checker for MISRA C++ 2008 rule 4-10-2-------------===//
```

Comment:
I'd like it to be the same as in all other source code files in LLVM/Clang: 80 characters

    reto@thinkpad:~/llvm/tools/clang/tools/clang-misracpp2008/src/rules$ grep -h "//===-  Rule_" *
    //===-  Rule_10_3_2.cpp - Checker for MISRA C++ 2008 rule 10-3-2-----------===//
    //===-  Rule_10_3_3.cpp - Checker for MISRA C++ 2008 rule 10-3-3-----------===//
    //===-  Rule_11_0_1.cpp - Checker for MISRA C++ 2008 rule 11-0-1-----------===//
    //===-  Rule_12_8_2.cpp - Checker for MISRA C++ 2008 rule 12-8-2-----------===//
    //===-  Rule_16_3_1.cpp - Checker for MISRA C++ 2008 rule 16-3-1-----------===//
    //===-  Rule_16_3_2.cpp - Checker for MISRA C++ 2008 rule 16-3-2-----------===//
    //===-  Rule_17_0_1.cpp - Checker for MISRA C++ 2008 rule 17-0-1-----------===//
    //===-  Rule_17_0_5.cpp - Checker for MISRA C++ 2008 rule 17-0-5 ----------===//
    //===-  Rule_18_0_1.cpp - Checker for MISRA C++ 2008 rule 18-0-1-----------===//
    //===-  Rule_18_0_2.cpp - Checker for MISRA C++ 2008 rule 18-0-2-----------===//
    //===-  Rule_18_0_3.cpp - Checker for MISRA C++ 2008 rule 18-0-3-----------===//
    //===-  Rule_18_0_4.cpp - Checker for MISRA C++ 2008 rule 18-0-4-----------===//
    //===-  Rule_18_0_5.cpp - Checker for MISRA C++ 2008 rule 18-0-5-----------===//
    //===-  Rule_18_2_1.cpp - Checker for MISRA C++ 2008 rule 18-2-1-----------===//
    //===-  Rule_18_4_1.cpp - Checker for MISRA C++ 2008 rule 18-4-1-----------===//
    //===-  Rule_18_7_1.cpp - Checker for MISRA C++ 2008 rule 18-7-1-----------===//
    //===-  Rule_19_3_1.cpp - Checker for MISRA C++ 2008 rule 19-3-1-----------===//
    //===-  Rule_2_10_1.cpp - Checker for MISRA C++ 2008 rule 2-10-1-----------===//
    //===-  Rule_2_10_2.cpp - Checker for MISRA C++ 2008 rule 2-10-2-----------===//
    //===-  Rule_2_13_3.cpp - Checker for MISRA C++ 2008 rule 2-13-3-----------===//
    //===-  Rule_2_13_4.cpp - Checker for MISRA C++ 2008 rule 2-13-4-----------===//
    //===-  Rule_2_13_5.cpp - Checker for MISRA C++ 2008 rule 2-13-5-----------===//
    //===-  Rule_27_0_1.cpp - Checker for MISRA C++ 2008 rule 27-0-1-----------===//
    //===-  Rule_3_3_1.cpp - Checker for MISRA C++ 2008 rule 3-3-1-------------===//
    //===-  Rule_3_3_2.cpp - Checker for MISRA C++ 2008 rule 3-3-2-------------===//
    //===-  Rule_3_4_1.cpp - Checker for MISRA C++ 2008 rule 3-4-1-------------===//
    //===-  Rule_3_9_2.cpp - Checker for MISRA C++ 2008 rule 3-9-2-------------===//
    //===-  Rule_4_5_1.cpp - Checker for MISRA C++ 2008 rule 4-5-1-------------===//
    //===-  Rule_5_14_1.cpp - Checker for MISRA C++ 2008 rule 5-14-1-----------===//
    //===-  Rule_5_18_1.cpp - Checker for MISRA C++ 2008 rule 5-18-1-----------===//
    //===-  Rule_5_8_1.cpp - Checker for MISRA C++ 2008 rule 5-8-1-------------===//
    //===-  Rule_6_2_1.cpp - Checker for MISRA C++ 2008 rule 6-2-1-------------===//
    //===-  Rule_6_2_2.cpp - Checker for MISRA C++ 2008 rule 6-2-2-------------===//
    //===-  Rule_6_2_3.cpp - Checker for MISRA C++ 2008 rule 6-2-3-------------===//
    //===-  Rule_6_3_1.cpp - Checker for MISRA C++ 2008 rule 6-3-1-------------===//
    //===-  Rule_6_4_1.cpp - Checker for MISRA C++ 2008 rule 6-4-1-------------===//
    //===-  Rule_6_4_2.cpp - Checker for MISRA C++ 2008 rule 6-4-2-------------===//
    //===-  Rule_9_5_1.cpp - Checker for MISRA C++ 2008 rule 9-5-1-------------===//